deploy/kubernetes/dolphinscheduler/templates/job-dolphinscheduler-schema-initializer.yaml

```diff
@@ -38,10 +38,12 @@ spec:
           image: {{ include "dolphinscheduler.image.fullname.tools" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
-            - bin/create-schema.sh
+            - tools/bin/upgrade-schema.sh
```

Comment:
I think we should add a variable to judge whether it is docker in ```upgrade-schema.sh```. If it is docker, we will not execute ```source dolphinscheduler-env.sh```. Otherwise, the no such file error will be printed to terminal which will misleading the user's judgment.

Reply:
Good suggestion. I had updated.